vendor/github.com/waku-org/go-waku/waku/v2/api/history/cycle.go

```diff
@@ -426,14 +406,30 @@ func (m *StorenodeCycle) WaitForAvailableStoreNode(ctx context.Context, timeout
 			select {
 			case <-m.StorenodeAvailableOneshotEmitter.Subscribe():
 			case <-ctx.Done():
+				if errors.Is(ctx.Err(), context.Canceled) {
+					return
```

Comment:
I wonder if we should log this condition

Reply:
Shouldn't be necessary, as a cancel would happen when you logout of the app.